diff --git a/rafx-api/src/backends/vulkan/swapchain.rs b/rafx-api/src/backends/vulkan/swapchain.rs
index 3fdb09fa..e80b1442 100644
--- a/rafx-api/src/backends/vulkan/swapchain.rs
+++ b/rafx-api/src/backends/vulkan/swapchain.rs
@@ -669,7 +669,7 @@ impl RafxSwapchainVulkanInstance {
             .image_usage(swapchain_image_usage_flags)
             .image_sharing_mode(vk::SharingMode::EXCLUSIVE)
             .pre_transform(surface_capabilities.current_transform)
-            .composite_alpha(vk::CompositeAlphaFlagsKHR::OPAQUE)
+            .composite_alpha(vk::CompositeAlphaFlagsKHR::POST_MULTIPLIED)
             .present_mode(present_mode)
             .clipped(true);
